modules/apps/object/object-rest-test/src/testFunctional/tests/BreakRelationshipBetweenCustomAndSystemObjects.testcase

```diff
@@ -161,7 +161,7 @@ definition {
 		property test.name.skip.portal.instance = "BreakRelationshipBetweenCustomAndSystemObjects#CanBreakOneToManyRelationshipWithCustomObjectAsParent";
 
 		task ("Given I relate the userAccount entry with the secondFoundation entry through the secondFoundation PUT endpoint") {
-			ObjectDefinitionAPI.relateObjectEntries(
+			var objects = ObjectDefinitionAPI.relateObjectEntries(
```

Comment:
Typo? Seems the var objects is not used in other places.

Reply:
oh, yes it should be removed, I only assigned it to print it for a debug. I removed the echo(objects) but forgot about themoving the actual var. thanks for catching this Ashley :)